lib/Pherkin/Extension/weasel_steps/widget_steps.pl

```diff
@@ -68,11 +61,13 @@
     my $label_text = $1;
     my $want_values = $2;
 
-    my $select = S->{ext_wsl}->page->find('*select', text => $label_text);
+    my @select = S->{ext_wsl}->page->find_all('*labeled', text => $label_text);
```

Comment:
I think you identified a nice hole in the select expander: it doesn't search for a label associated with the select widget.
However, this isn't a solution: you now may match anything that's not a select/dropdown widget.